lib/ASRouter.jsm

```diff
@@ -553,6 +562,9 @@ class _ASRouter {
       case "IMPRESSION":
         this.addImpression(action.data);
         break;
+      case "INSTALL_ADDON_FROM_URL":
```

Comment:
you need to add this to `ASRouterActions` in Actions.jsm (see `ASRouterUtils.executeAction` in asrouter-content). This is for security reasons (i.e. so you can't just specify any action in a message and have it work)